src/docfx.website.themes/default/partials/class.tmpl.partial

```diff
@@ -33,11 +33,11 @@
 {{#docurl}}
 <span class="small pull-right mobile-hide">
   <span class="divider">|</span>
-  <a href="{{docurl}}">Improve this Doc</a>
+  <a href="{{docurl}}"><span class="fa fa-github"></span>Improve this Doc</a>
```

Comment:
Let's hold changes to the styles from this change list.
It seems too aggressive to change the default template. It could be better to overwrite the template in our own Documentation/template.